app/views/users/show.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
```

Comment:
I would recommend this to have a `gravatar_for` helper method in `ApplicationHelper` you will re-use it.